### PR TITLE
created models and seeders

### PR DESCRIPTION
**Pull request by SreenathES**

### CODE REVIEW

File: server/config/db.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('my_learn', 'root', 'experion@123', {
```
Comment (mishelshaji) [inline server/config/db.config.js]: Try to read these data from an env file.

File: server/models/course.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        allowNull: false
+    },
+
+    level: {
+        type: DataTypes.STRING(15),
+        allowNull: false
+    },
+
+    price: {
+        type: DataTypes.FLOAT,
```
Comment (mishelshaji) [inline server/models/course.js]: Use decimal instead.

File: server/models/course.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+const Course = sequelize.define('course', {
+    id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+    },
+
+    title: {
+        type: DataTypes.STRING,
```
Comment (mishelshaji) [inline server/models/course.js]: Specify a size.

File: server/models/otp.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+const User = require('./user');
+
+const Otp = sequelize.define('otp', {
+    id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+    },
+
+    otp_code: {
```
Comment (mishelshaji) [inline server/models/otp.js]: change the column name to otp.

File: server/models/payment.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        autoIncrement: true
+    },
+
+    payment_method: {
+        type: DataTypes.STRING(10),
+        allowNull: false
+    },
+
+    amount: {
+        type: DataTypes.FLOAT,
```
Comment (mishelshaji) [inline server/models/payment.js]: Change to decimal.

File: server/models/role.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+const Role = sequelize.define('role', {
+    id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+    },
+
+    role: {
+        type: DataTypes.STRING(10),
+        allowNull: false
```
Comment (mishelshaji) [inline server/models/role.js]: Make unique.

File: server/models/userCourseEnrollment.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+        primaryKey: true,
+        autoIncrement: true
+    },
+
+    enrollment_date: {
+        type: DataTypes.DATE,
+        allowNull: false
+    },
+
+    enrollment_status: {
```
Comment (mishelshaji) [inline server/models/userCourseEnrollment.js]: Why not use status instead as the column name?

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: server/models/course.js (hunk 2 above)
Comment (SreenathES) [reply server/models/course.js]: issue resolved

File: server/models/otp.js (hunk 4 above)
Comment (SreenathES) [reply server/models/otp.js]: name changed to otp

File: server/models/payment.js (hunk 5 above)
Comment (SreenathES) [reply server/models/payment.js]: Datatype changed to decimal.

File: server/models/role.js (hunk 6 above)
Comment (SreenathES) [reply server/models/role.js]: issue resolved

File: server/models/userCourseEnrollment.js (hunk 7 above)
Comment (SreenathES) [reply server/models/userCourseEnrollment.js]: Changed column name from enrollment_status to status.

File: server/config/db.config.js (hunk 1 above)
Comment (SreenathES) [reply server/config/db.config.js]: Issue resolved.

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**